src/main/features/claim/views/total.njk

```diff
@@ -0,0 +1,55 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, saveAndContinueButton %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('Your issue fee') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form novalidate method="post" class="form-group">
+        {{ csrfProtection(csrf) }}
+
+        <table class="table-form form-group">
```

Comment:
don't use a table. use divs. I'll be pushing changes to the prototype soon that will show you how to do it without a table